docs/README.linux

```diff
@@ -57,12 +57,15 @@ Build-Depends: autoconf, automake, autopoint, autotools-dev, cmake, curl,
   libtag1-dev (>= 1.8), libtiff-dev, libtinyxml-dev (>= 2.6.2), libtool, libudev-dev,
   libusb-dev, libva-dev, libvdpau-dev, libvorbis-dev, libxinerama-dev, libxml2-dev,
   libxmu-dev, libxrandr-dev, libxslt1-dev, libxt-dev, libyajl-dev (>=2.0), lsb-release,
-  nasm [!amd64], python-dev, python-imaging, python-support, swig, unzip, yasm, zip,
-  zlib1g-dev, uuid-dev
+  nasm [!amd64], python-dev, python-imaging, python-support, swig, unzip, uuid-dev, yasm, zip,
+  zlib1g-dev
+  
+[NOTICE] crossguid / libcrossguid-dev all Linux destributions.
+Kodi now requires crossguid which is not available in Ubuntu repositories at this time.
+See Readme.ubuntu
```

Comment:
please remove the ubuntu reference here, this is a general linux readme.
Instead please add the manual crossguid compile instruction